eclipse/sw360
# feat(Obligation): Add License Obligation from License Database based on licenses found in accepted attachments in Release and its LicenseInfo attachmentUsage in Project

> Add License Obligation from License Database based on licenses found in accepted attachments in Release and its LicenseInfo attachmentUsage in Project
> * Which issue is this pull request belonging to and how is it solving it? (*#1046*)
> * Did you add or update any new dependencies that are required for your change?

### How To Test?
> - Add releases to project with approved License info attachment . 
> - Select attachmentusage for that Release as LicenseInfo in Project.
>    - popup msg should appear if above two prerequisite not satisfied
> - Edit Project -> Obligation tab -> click on "Add License Obligation".
> - It should fetch obligation from License DB based on Licenses found in LicenseInfo attachment.
> - Add, Update , Delete orphaned obligation
![Capture](https://user-images.githubusercontent.com/56534710/99506255-e13bc500-29a7-11eb-934f-f644ebe5dfe1.JPG)

![image](https://user-images.githubusercontent.com/56534710/99505874-62468c80-29a7-11eb-8b3e-74a8cca68374.png)

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
